tools/theme/README.md

```diff
@@ -1,271 +1,99 @@
-## Description
+## Overview
 
-`sp-theme` applies a Spectrum theme by using CSS custom properties to set default sizes & colors for all of the components in its scope. The Spectrum design system provides four color themes (`lightest`, `light`, `dark`, and `darkest`) and two different scales (`medium` and `large`) to support desktop & mobile UI.
+`<sp-theme>` provides Spectrum design tokens (CSS custom properties) to everything in its DOM scope. Components inside a theme use these tokens to render correctly. The element itself does not visually “apply” styles to your app; it exposes the tokens so your components (and any of your CSS) can consume them.
+
+Spectrum offers multiple variants:
```

Comment:
nit: I see the API tab for Theme is not surfacing the properties / attributes. I recommend looking into it if it is a quick fix.